### ISSUE TITLE
Proxy Support

### ISSUE THREAD

**Issue by stplaydog** [ckreibich/scholar.py#54]
I can not connect to google scholar using this file because of the network issues in our company.
If this program can add proxy support, it will be the best.

**Comment by YborBorn** [comment]
I agree.  I'd love to use this but am behind a proxy as well.  Admittedly, I'm a newbie when it comes to RESTful calls, but in the past I've assigned my proxy to a variable and called that variable in a requests.get() call.  I'm thinking the procedure would be similar for scholar.py.  Here's what I've tried, which has worked for me with other python code, but not with scholar.py:  
1. Create a global proxy variable:
proxies = {
  'http': 'http://myProxyHere.com',
  'https': 'http://myProxyHere.com'
} 
2.  In the "def _get_http_response(self, url, log_msg=None, err_msg=None)" method, I edited the "req" call: 
            req = Request(url=url,
                          proxies=proxies,
                          headers={'User-Agent': ScholarConf.USER_AGENT},
                          verify=False)

Alas, this does not work.  If anyone can help here, it would surely be appreciated.
